docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -172,6 +175,23 @@ To create an export configuration, do the following:
     http://<victoria-metrics-endpoint-host-address>/opentelemetry
     ```
 
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="GCS" lang="gcs" %}}
```

Comment:
We mention `Google Cloud Storage` everywhere in UI. So better to keep the same in docs as well?

Reply:
Well the tabs were getting a bit wide on the screen

Reply:
Can we change in UI - **_Google Cloud Storage (GCS)_** ?

Reply:
No no, in docs I will add "(GCS)" at first occurrance - this is standard for documentation